server/src/controllers/Venue/resolver.ts

```diff
@@ -102,6 +104,85 @@ export class VenueResolver {
     @Arg('_onlyUsedForAuth', () => Int) _onlyUsedForAuth: number,
   ): Promise<{ id: number }> {
     // TODO: handle deletion of non-existent venue
+    const users = await prisma.users.findMany({
+      where: {
+        user_events: {
+          some: {
+            event: {
+              canceled: false,
+              ends_at: { gt: new Date() },
+              venue_id: id,
+            },
+            subscribed: true,
+            attendance: { name: 'yes' },
+          },
+        },
+      },
+      include: {
+        user_events: {
+          include: { event: true },
+          where: {
+            event: {
+              canceled: false,
+              ends_at: { gt: new Date() },
+              venue_id: id,
+            },
+            subscribed: true,
+            attendance: { name: 'yes' },
+          },
+        },
+      },
+    });
+
+    const venue = await prisma.venues.findUniqueOrThrow({
+      where: { id: id },
+    });
+
+    const eventList = (event: events[], currentUserId: number) => {
+      event.map(
+        ({ name, id }) => `
+      <tr>
+      <td>${name}</td>
+      <td>${eventUnsubscribeOptions({
+        eventId: id,
+        userId: currentUserId,
+      })}</td>
+     </tr>
+  `,
+      );
+    };
```

Comment:
```suggestion
    const eventList = (event: events[], currentUserId: number) =>
      event.map(
        ({ name, id }) => `
      <tr>
        <td>${name}</td>
        <td>${eventUnsubscribeOptions({
          eventId: id,
          userId: currentUserId,
        })}</td>
      </tr>
      `,
      );
```
Resulting array wasn't returned.

Reply:
Do we add the eslint rule, that warns about missing `return` in the functions 🤔?

Btw, how did you test it? I am asking because, I can't think of a way to trigger the resolver to test it, that isn't creating a button and requesting the resolver to trigger. https://github.com/freeCodeCamp/chapter/pull/2192